gitpod-io/gitpod
# flag: add BETA Label to `/tokens` page

> Signed-off-by: Siddhant Khare <siddhant@gitpod.io>

## Description
This PR adds a Beta Label to `/tokens` page.

|Preview (Light)|Preview(Dark)|
|---|---|
|![image](https://user-images.githubusercontent.com/55068936/206161919-9d832b90-cc48-49db-b3c5-c45e59bf1e49.png)|![image](https://user-images.githubusercontent.com/55068936/206162533-c78bf745-f5d6-43e8-8f3c-2ab637c31ae6.png)|



## Related Issue(s)

Fixes #[internal conversation](https://gitpod.slack.com/archives/C0463A2KQUU/p1670333310895679?thread_ts=1670333253.863319&cid=C0463A2KQUU)

## How to test

- Open Preview
- Go To `/tokens` & see :eyes: (if feature flag is enable for you)

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
